test/dev/dev.test.ts

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+
+const execa = require('execa');
+const {readdirSync, readFileSync, statSync, existsSync} = require('fs');
+const glob = require('glob');
+const os = require('os');
+const got = require('got');
+
+describe('snowpack dev', () => {
+  // tests don't run on windows. `snowpackProcess` does not get ended correctly
+  if (process.platform === 'win32') {
+    it.todo('snowpack dev tests are currently not running on Windows');
+    return;
+  }
```

Comment:
Tests are now failing because jest complains about incorrect snapshots ...

Reply:
yup, can't do this with snapshot testing, unfortunately